float32 position
bool busy
bool holding
bool in_position
float32 current_load